content/guides/building/get-source-git.md

```diff
@@ -1,15 +1,20 @@
 +++
 type = "article"
-title = "Getting the source code"
+title = "Get the Haiku Source Code / Pushing Patches to Haiku 101"
 date = "2011-06-13T16:07:27.000Z"
 tags = []
 +++
 
+ <!-- NOTE: This article should be eventually split in two parts. -->
+
 <span class="right"><img src='/images/archive_64.png'></span>
 
 Haiku's source code is currently being hosted in a <a href="http://git-scm.com/" target="_blank">Git based repository</a> with <a href="https://gerritcodereview.com">Gerrit code review</a>.
 Anonymous access will allow anyone to download Haiku's source code; However, the authenticated (non-anonymous) method is required for submitting patches.
 
+Although most contributors tend to disclose their real names, we do not currently have a real name policy and
```

Comment:
```suggestion
Although most contributors tend to disclose their real names, we do not have a real name policy and
```

(the "currently" implies that we are considering changing this)

Also, we tend to refer to people by their peudonyms and not realnames anyway. So let's just say that's the main thing we use.

Your wording here sounds like it's realname by default, while it isn't.